addons/web/static/src/scss/bootstrap_review.scss

```diff
@@ -80,6 +80,18 @@
     }
 }
 
+// Do not display the backdrop element added by bootstrap in the body and added a
+// background on the modal to keep the same effect. The bootstrap backdrop was useful
+// for compatibility with <IE9 but is no longer needed. This also avoids z-index
+// issues because the wrapwrap (z-index: 0) can contain a modal. And in this case,
+// the backdrop (z-index: 999) goes above the modal.
+.modal-backdrop {
+    display: none;
+}
+.modal:not([data-backdrop="false"]) {
+    background-color: rgba($modal-backdrop-bg, $modal-backdrop-opacity);
+}
+
```

Comment:
Actually I was wrong to add a bg-color only if data-backdrop was "true" or "static". We need it it even there is no data-backdrop attribute. So the only condition for not having a background is when data-backdrop is "false".